src/css/App.css

```diff
@@ -302,6 +304,15 @@ dd {
   #playContainer::before {
     display: none;
   }
+  .animation {
+    background-image: url('https://cdn-media-1.freecodecamp.org/code-radio/bg.jpg');
```

Comment:
This is a large image. Is there not a smaller version we could use @ahmadabdolsaheb ?

Reply:
We could load different images for different screen sizes, but we are only loading the background image for large screens any way.

Reply:
Right, but with this line, we are loading a 1920x1013 image for screens less than 750px wide

Reply:
Maybe load a blue screen for screens less than 750 px wide ? Like what we do for screens less than 500 px ?